gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java

```diff
@@ -193,7 +193,7 @@ protected void configureForSourceSet(final SourceSet sourceSet, SpotBugsTask tas
             public FileCollection call() {
                 // the simple "classes = sourceSet.output" may lead to non-existing resources directory
                 // being passed to SpotBugs Ant task, resulting in an error
-                return project.fileTree(sourceSet.getOutput().getClassesDirs()).builtBy(sourceSet.getOutput());
+                return project.fileTree(sourceSet.getOutput().getClassesDir()).builtBy(sourceSet.getOutput());
```

Comment:
@iloveeclipse could you invite following user to this spotbugs organization? Then we can make message form SonarQube more intuitive.
https://github.com/spotbugs-bot